### ISSUE TITLE
Dont Fade + Don't Enter Bug

### ISSUE THREAD

**Issue by BrownieGamingX5** [absolllute/Mega-Hack-Pro-Future#1121]
Sometimes when objects are scaled up a lot, even with "Don't Fade" and "Don't Enter" enabled,  it glitches out and doesn't even load in-game. This is really annoying when having to deal with largely scaled up objects disappearing in-game when they aren't supposed to. It also causes visual bugs too, which can sometimes be very noticeable.

**Comment by MainConMan** [comment]
Ive experienced this too, it might be a bug with vanilla gd, but not sure

**Comment by NewAlexGamer** [comment]
yes if objects are scaled by a large amount, usually going above 5, they start flashing and doing weird stuff
